Tests/mocks/Wraps.cpp

```diff
@@ -9,3 +10,15 @@ extern "C" FILE* __wrap_popen(const char* command, const char* type)
 {
     return Wraps::getInstance().popen(command, type);
 }
+
+//This function will be called for syslog() in the code (added -Wl,-wrap,syslog)
+extern  "C" void __wrap_syslog(int pri, const char *fmt, ...)
```

Comment:
this should call real syslog as well

Reply:
What is the need for calling real syslog in this test perspective?

Reply:
Done